packages/app-mobile/components/ScreenHeader/index.tsx

```diff
@@ -419,6 +421,28 @@ class ScreenHeaderComponent extends PureComponent<ScreenHeaderProps, ScreenHeade
 			);
 		};
 
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
+		const renderTogglePluginEditorButton = (styles: any, onPress: OnPressCallback, disabled: boolean) => {
+			if (!this.props.showPluginEditorButton) return null;
+
+			return (
+				<IconButton
+					onPress={onPress}
+					disabled={disabled}
+
+					themeId={themeId}
+					description={_('Toggle plugin editor')}
+					accessibilityHint={
+						disabled ? null : _('Toggle plugin editor')
+					}
```

Comment:
The `accessibilityHint` is generally read after the `description` and is intended to add extra context. It should be fine to omit.
```suggestion
```